### DISCUSSION TITLE
rtl-airband + rdio-scanner with scan mode TGs (TalkGroups)

### DISCUSSION THREAD

**Discussion by Brubeck48** [rtl-airband/RTLSDR-Airband#309] [Q&A]
Hi!

I would like to use scan mode in rtl-airband + rdio-scanner setup. But there is a challenge how to give for each scanned frequencies specific talkgroups ids? Any ideas?

For example the name of each files should be: cymx_#SYS_#TG_#DATE_#TIME_#HZ. 

Thank you

```
type = "rtlsdr";
serial = "00000102";
gain = 38.7;
mode = "scan";
channels:
    (
      {
        freqs = ( 127000000, 12800000, 12900000 );
        modulation = "am";
        outputs: (
          {
            type = "file";
            directory = "/air";
            filename_template = "cymx_4";
            split_on_transmission = true;
            include_freq = true;
          }
        );
      }
    );
  }
```

**Comment by charlie-foxtrot** [comment]
This question may be better asked to `rdio-scanner` but based on this comment, it may have something to do with how you configure `dirWatch`? https://github.com/szpajder/RTLSDR-Airband/pull/163#issuecomment-647544955

**Comment by Brubeck48** [comment]
Thank you @charlie-foxtrot for your quick answer! 

I will try to specify what I mean. Now we have a structure of filenames according to your comment in the parallel thread: if `split_on_transmission, include_freq` enabled the output files will be like: some text_date_time_frequency.mp3

If I use multichannel mode everything is ok. I can specify manually talk group for each channel frequency by adding TGs ID to the prefix "some text", for example dispatch_id of talkgroup... But when I use scan mode I can add such prefix only for all files which will be created from frequency list. 

I'm looking for a way to create files with specific TG ID for each specific frequency in scan mode. For example: 
lets imagine that frequency:
1. 127.000 has id 1
2. 128.000 has id 2
3. 129.000 has id 3

So the output files for config from my fist comment should be:
1) cymx_4_(talkgroupid)**1**_20200621_151750_127000000.mp3
2) cymx_4_**2**_20200621_151750_128000000.mp3
3) cymx_4_**3**_20200621_151750_129000000.mp3

Which will be correspond with dirwatch structure for monitored folder in the Rdio-scanner settings: 
cymx_#SYS_#TG_#DATE_#TIME_#HZ

How to assign specific id for each frequency in scan mode?

**Comment by charlie-foxtrot** [comment]
dont think there is a way to do that from within `rtl-airband`.  maybe have a script watching the `rtl-airband` output directory then do a rename / move to `rdio-scanner` input directory?
